platform.md

```diff
@@ -566,6 +567,10 @@ Usage:
       - All run-image layers SHALL be preserved
       - All run-image config values SHALL be preserved unless this conflicts with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
+    - MUST contain a layer containing all buildpack-provided standardized Bill of Materials (BOM) files for `launch` as determined by the [Buildpack Interface Specfication](buildpack.md)
+      - `<layers>/BOM/<buildpack-id>/launch.bom.<ext>` MUST contain the buildpack-provided `launch` BOM
```

Comment:
Does this mean that this would cause issues if a buildpack with buildpack id `bom` exists? (We currently allow such buildpack ids)

Reply:
The buildpack would need id `sbom`, but yes